DFC.Digital/DFC.Digital.AcceptanceTest/Infrastructure/Pages/JobProfilePage.cs

```diff
@@ -13,7 +13,7 @@ public class JobProfilePage : SitefinityPage<JobProfileDetailsViewModel>
 
         public int NumberOfRelatedCareers => Find.Elements(By.CssSelector(".list-big li")).Count();
 
-        public bool HasHowToBecomeSection => Find.OptionalElement(By.Id("HowToBecome")) != null;
+        public bool HasHowToBecomeSection => Find.OptionalElement(By.Id("how-to-become")) != null;
```

Comment:
I think this is incorrect, the HTB widget was also changed so no need to change this ID.

Reply:
Changes made to the ID